plugins/wp-songs-library/includes/class-wp-songs-library.php

```diff
@@ -270,6 +278,12 @@ private function define_custom_tables() {
 		$this->loader->add_action( 'plugins_loaded', $plugin_custom_table, 'register_custom_tables' );
 	}
 
+	private function define_cli_command() {
+		if ( class_exists( 'Wp_Songs_Library_Cli' ) ) {
+			$plugin_cli = new Wp_Songs_Library_Cli( $this->get_plugin_name(), $this->get_version() );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$plugin_cli" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://54.213.144.90:9000/sonar/coding_rules#rule_key=php%3AS1481)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable "$plugin_cli" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://54.213.144.90:9000/sonar/coding_rules#rule_key=php%3AS117)